JSDemos/Demos/Charts/PieWithCustomStyles/description.md

```diff
@@ -0,0 +1,47 @@
+This demo shows how to implement custom patterns and gradients in the PieChart component points.
+
+## Gradient
+
+Call the [registerGradient()](/Documentation/ApiReference/Common/Utils/viz/#registerGradienttype_options) method to obtain a gradient id. Specify method arguments:
+
+- `type`    
+This method supports two gradient types: *'linear'* and *'radial'*.
+
+- `options`    
+An object with gradient options.
+
+    - `colors`    
+    An array of gradient colors. Each color is an object that includes `color` and `offset` fields. The `offset` field specifies a starting point for each color.
+
+    - `rotationAngle` (linear gradients only)
+    You can use this option to rotate linear gradients.
+
+## Pattern
+
+To acquire a pattern id, implement the [registerPattern()](/Documentation/ApiReference/Common/Utils/viz/#registerPatternoptions) method. Specify method arguments:
```

Comment:
If I understand correctly, they don't need to "implement" the method. They only need to call it. 

```suggestion
To acquire a pattern ID, call the [registerPattern()](/Documentation/ApiReference/Common/Utils/viz/#registerPatternoptions) method. Specify method arguments:
```